src/elm/Page/History.elm

```diff
@@ -209,13 +248,27 @@ viewOrder shared model order =
                             , List.indexedMap
                                 (viewTravelRight shared (List.length order.travelRights))
                                 order.travelRights
-                            , [ H.div [ A.style "display" "flex" ]
+                            , [ H.div
+                                    ([ A.style "display" "flex"
```

Comment:
If I'm reading this correctly, I think this should be a button.

```suggestion
                            , [ H.button
                                    ([ A.type "button"
                                    , A.style "display" "flex"
```

Reply:
I'm not sure whether this would be considered a button or not, it's certainly not styled as such. The `button` element carries a bunch of default styles, but I think I got rid of most of them.

Reply:
Button styling/linking is something I can look into as part of atb-as/theme also. But we have to consider semantics here, not styling. If something triggers an action not leading to URL change -> `button`. If it's navigational (action causing URL change only without side-effects) -> `a`. Both can be styled as different buttons or not, but typically buttons don't have pointer cursors.

Reply:
I agree, but there's no indication that this is something you can interact with. Buttons by default at least look like buttons, and links are underlined by default. This is why I set pointer cursor, to at least give _some_ feedback to the user that this is something you can interact with. I think we should really go through this with design so we all agree on how this is supposed to work. There _are_ buttons in the design, with a cyan background, but these are different. I suppose these might be intended to be navigational items, but it's not clear to me at least.

I'll merge this as it has little to do with the PR itself, and we need to go through the design, CSS, and semantics for the entire app.